site/lib/getBlockRate.ts

```diff
@@ -0,0 +1,36 @@
+import { POLYGONSCAN_API_KEY, MONTH_IN_SECONDS } from "lib/constants";
+import { getJsonRpcProvider } from "@klimadao/lib/utils";
+
+// Calculates block rate by dividing one month in seconds by number of blocks that got created during that month
+
+export async function getBlockRate() {
```

Comment:
protip for vscode, if you use JSDoc style comments, then they appear in the hover-preview when you hover the variable elsewhere in the codebase

```suggestion

/** Calculates block rate by dividing one month in seconds by number of blocks that got created during that month */
export async function getBlockRate() {
```